docs/modules/reference/pages/daemons/daemon-config-files/reportd.adoc

```diff
@@ -1,17 +1,16 @@
+
 [[ref-daemon-config-files-reportd]]
 = Reportd
 
-The report daemon handles database report generation and scheduling.
+The reportd daemon handles database report generation and scheduling.
```

Comment:
We are inconsistent across these files. Sometimes we say "The X Y Z daemon", other times we say "the Xd daemon", still others we just define it "Generates a ...". Not necessary to do as part of this Jira, but we should standardize the intro text. 

I like spelling it out "The Passive Status Keeper daemon" or "The report daemon ..." "reportd daemon" is redundant, since the "d" means daemon. There are a lot of instances of this in the docs, so again, not necessary to address that now.

Reply:
I'll be sure to note this for later!